hnn_core/network.py

```diff
@@ -391,6 +392,9 @@ def _create_all_spike_sources(self):
                 pc.cell(gid, self.cells[-1].connect_to_target(
                         None, self.params['threshold']))
 
+        for feed in self.feeds:
```

Comment:
So our goal here is to set spike inputs in their own loop (i.e., separate from looping through `self._gid_list`)? Why is this necessary?

Also, `self.feeds` should be a list of both `self.feeds_common` and `self.unique_feeds` right?